desafios/modulo-01/d005/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="pt-br">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Social</title>
+</head>
+<body>
+    <h1>Minhas redes sociais</h1>
+    <h2>Quem sou eu?</h2>
+    <img src="imagens/foto-perfil-pedro.png" alt="imagem.png">
+    <p>Meu é <strong>Pedro Henrique</strong>,sou um garoto de 16 anos e estou gostando muito de atuar como desevolvedor,e gosto muito também de cuidar da munha saude fazendo exercicios fisico e praticar esportes radicais como parkur,surf,andar de bicicleta,arte marcial e gosto tambem de fazer esportes como futebol,natação e outros... Comecei a desenvolver site na web por um simples motivo, sempre gostei de aprender, sempre tive o dom de adquerir conecimento e essa area de desenvolvimento programação, são multmilionaria em conhecimento, estou aprendendo o básico ainda, mais sei que daqui um tempo estudando e sempre me atualizando vou estar dominando essa função.</p>
```

Comment:
Bacana sua fome por conhecimento Pedro!
Segue aqui algumas sugestões de melhorias de português no seu texto:
```suggestion
    <p>Meu nome é <strong>Pedro Henrique</strong>, sou um garoto de 16 anos e estou gostando muito de atuar como desenvolvedor. Gosto muito, também, de cuidar da minha saúde fazendo exercícios físicos e praticando esportes radicais como parkur, surf, andar de bicicleta, artes marciais... gosto, também, de fazer esportes como futebol, natação e outros...</p>
   <p>Comecei a desenvolver sites na web por um simples motivo, sempre gostei de aprender, sempre tive o dom de adquirir conhecimento, e essa área de desenvolvimento/programação é multimilionária em conhecimento. Estou aprendendo o básico ainda, mais sei que daqui um tempo estudando e sempre me atualizando vou estar dominando essa função.</p>
```